wpiutil/CMakeLists.txt

```diff
@@ -100,7 +100,7 @@ file(GLOB_RECURSE wpiutil_windows_src src/main/native/windows/*.cpp)
 file(GLOB fmtlib_native_src src/main/native/thirdparty/fmtlib/src/*.cpp)
 file(GLOB_RECURSE memory_native_src src/main/native/thirdparty/memory/src/*.cpp)
 
-add_library(wpiutil ${wpiutil_native_src} ${fmtlib_native_src} ${memory_native_src} ${wpiutil_resources_src})
+add_library(wpiutil ${wpiutil_native_src} ${memory_native_src} ${wpiutil_resources_src})
```

Comment:
That breaks builds that don't use a system install of fmtlib.

Reply:
the target_sources line below doesn't address that issue?

Reply:
It is added here: 
https://github.com/wpilibsuite/allwpilib/blob/d37f990ce3a8a36bc791dd989dcea2868759e556/wpiutil/CMakeLists.txt#L121-L122